examples/pytorch/graphsage/link_pred.py

```diff
@@ -11,6 +12,18 @@
 # (This is a long-standing issue)
 from ogb.linkproppred import DglLinkPropPredDataset
 
+class ArgParser(argparse.ArgumentParser):
+    def __init__(self):
+        super(ArgParser, self).__init__()
+        self.add_argument('--pure-gpu', action='store_true',
+                          help='mix CPU and GPU training')
+
+    def parse_args(self):
+        args = super().parse_args()
+        return args
```

Comment:
Why are we subclassing `ArgumentParser`?  I think what people usually do is
```python
parser = argparse.ArgumentParser()
parser.add_argument(...)
args = parser.parse_args()
```

Three lines of code.

Reply:
Done